### PR TITLE
Test rendering of ExpressionEditorColumn

### PR DESCRIPTION
**Pull request by jomarko**
@manstis please have a look

### CODE REVIEW

**Comment by manstis** [comment]
@jomarko Hi could you please rebase? 

Roger made a big change with a PR on ```master``` that has been merged into my fork.

**Comment by jomarko** [comment]
@manstis Rebased.

File: kie-wb-common-dmn/kie-wb-common-dmn-client/src/test/java/org/kie/workbench/common/dmn/client/editors/expressions/types/context/ExpressionEditorColumnRendererTest.java [hunk 1]
```diff
@@ -0,0 +1,67 @@
+package org.kie.workbench.common.dmn.client.editors.expressions.types.context;
```
Comment (manstis) [inline kie-wb-common-dmn/kie-wb-common-dmn-client/src/test/java/org/kie/workbench/common/dmn/client/editors/expressions/types/context/ExpressionEditorColumnRendererTest.java]: License please!

**Comment by jomarko** [comment]
@manstis done

**Pull request merged**